tests/suites.c

```diff
@@ -331,8 +338,8 @@ static int execute_test_case(int svr_argc, char** svr_argv,
             printf("server command line too long\n");
             break;
         }
-        strcat(commandLine, svr_argv[i]);
-        strcat(commandLine, flagSep);
+        wc_strlcat(commandLine, svr_argv[i], sizeof commandLine);
```

Comment:
Can we setup a `XSTRLCAT` macro and have it map to `wc_strlcat` by default? We do this for a few others with `STRING_USER` allowing people to provide their own std-lib. Same for `wc_strlcpy`.

Reply:
yes great idea, will do.